app/resources/v1/streets_pg.js

```diff
@@ -9,14 +9,9 @@ const Op = Sequelize.Op
 
 exports.post = async function (req, res) {
   let body
-  const street = new Street()
-
-  // we are mirroring a mongoDB call, so use the same IDs.
-  const hasPriorCall = !!(
-    res.mainData &&
-    (res.mainData.id || res.mainData.status)
-  )
-  street.id = hasPriorCall ? res.mainData.id : uuid.v1()
+  const street = {}
+
+  street.id = uuid.v1()
```

Comment:
Looks like we need to standardize on importing `uuidv1` from `uuid/v1` throughout.